api/src/main/java/org/pmiops/workbench/google/CloudStorageServiceImpl.java

```diff
@@ -112,38 +111,47 @@ public void writeFile(String bucketName, String fileName, byte[] bytes) {
     storage.create(blobInfo, bytes);
   }
 
+  private JSONObject getCredentialsBucketJSON(final String objectPath) {
+    return new JSONObject(readCredentialsBucketString(objectPath));
+  }
+
   @Override
   public JSONObject getJiraCredentials() {
-    return new JSONObject(readToString(getCredentialsBucketName(), "jira-login.json"));
+    return getCredentialsBucketJSON("jira-login.json");
   }
 
   @Override
   public JSONObject getElasticCredentials() {
-    return new JSONObject(readToString(getCredentialsBucketName(), "elastic-cloud.json"));
+    return getCredentialsBucketJSON("elastic-cloud.json");
+  }
+
+  private String readCredentialsBucketString(String objectPath) {
+    return readToString(getCredentialsBucketName(), objectPath);
+  }
+
+  private GoogleCredential getBucketCredential(final String objectPath) throws IOException {
```

Comment:
I'd just call this `getCredential`. I typically don't put `final` in argument lists. I just hunt down anyone who reassigns a passed-in argument and hit them with a rolled-up newspaper.